src/saveload/cheat_sl.cpp

```diff
@@ -33,6 +33,7 @@ static const SaveLoad _cheats_desc[] = {
 	SLE_VAR(Cheats, setup_prod.value, SLE_BOOL),
 	SLE_VAR(Cheats, edit_max_hl.been_used, SLE_BOOL),
 	SLE_VAR(Cheats, edit_max_hl.value, SLE_BOOL),
+	SLE_VAR(Cheats, station_rating.value, SLE_BOOL),
```

Comment:
Shouldn't this be `SLE_CONDVAR()` with a savegame bump?

You are also not loading/saving `been_used` here.

Reply:
I...actually don't know. I've never fully understood what SLE_CONDVAR() does. But the `been_used` checkbox works now. 😉

Reply:
It makes the variable CONDitional based on savegame version.

Savegames from before this cheat is added won't have this variable, so it needs to be included only for new games, and the savegame bumped.

Reply:
Ah, so when loading an older game, saveload won't look for a variable that doesn't exist yet, finding something else in the memory where it should be?